### DISCUSSION TITLE
Why can't extensions use the `log` library?

### DISCUSSION THREAD

**Discussion by valentinegb** [zed-industries/zed#11472] [Feature Requests]
I tried it in an extension of mine and it built seemingly just fine, but when I run Zed through my terminal with `RUST_LOG=debug`, I see neither `warn` nor `debug` messages from my extension. Why is this? Is this something that can be changed? I would not consider including `println`s in released versions of my extension, so that means I would basically just have to delete any debug messages before releasing.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
